src/main/kotlin/gg/skytils/skytilsmod/core/GuiManager.kt

```diff
@@ -236,13 +238,15 @@ object GuiManager : PersistentSave(File(Skytils.modDir, "guipositions.json")) {
     }
 
     override fun read(reader: Reader) {
-        json.decodeFromString<Map<String, GuiElementLocation>>(reader.readText()).forEach { name, (x, y, scale) ->
+        json.decodeFromString<Map<String, GuiElementLocation>>(reader.readText()).forEach { name, (x, y, scale, textShadow) ->
             val pos = x to y
             GUIPOSITIONS[name] = pos
             GUISCALES[name] = scale
+            GUITEXTSHADOWS[name] = textShadow // TODO: Fix/Work around NullPointerException with old guipositions file
```

Comment:
comment resolved?

Reply:
No.. this will crash if you have a config file from a previous version